CMakeLists.txt

```diff
@@ -162,6 +162,7 @@ option(USE_OPENCV "Use OpenCV" OFF)
 option(USE_OPENMP "Use OpenMP for parallel code" ON)
 option(USE_PROF "Use profiling" OFF)
 option(USE_QNNPACK "Use QNNPACK (quantized 8-bit operators)" ON)
+option(USE_PYTORCH_QNNPACK "Use QNNPACK (quantized 8-bit operators)" ON)
```

Comment:
We probably can remove this and set USE_PYTORCH_QNNPACK on line #300 (in INTERN_BUILD_MOBILE AND NOT BUILD_CAFFE2_MOBILE). pytorch_qnnpack will be merged into libtorch as default quantization engine for mobile build so I feel we don't have to expose the option to users.